GoogleSignIn/Sources/GIDSignInInternalOptions.h

```diff
@@ -41,6 +43,9 @@ NS_ASSUME_NONNULL_BEGIN
 /// Whether the sign-in is an addScopes flow. NO means it is a sign in flow.
 @property(nonatomic, readonly) BOOL addScopesFlow;
 
+/// The user account details the Verify with Google flow will verify
+@property(nonatomic, copy, nullable, readonly) NSArray<GIDVerifiableAccountDetail *> *accountDetailsToVerify;
```

Comment:
Everything related to account detail verification needs to be enclosed with `#if TARGET_OS_IOS`.

Reply:
Done.